ConfigSpace/configuration_space.py

```diff
@@ -45,20 +45,32 @@
 
 
 class ConfigurationSpace(object):
+
     # TODO add comments to both the configuration space and single
     # hyperparameters!
 
     # TODO add a method to add whole configuration spaces as a child "tree"
 
-    """Represent a configuration space.
-    """
-
     def __init__(
             self,
             name: Union[str, None] = None,
             seed: Union[int, None] = None,
             meta: Optional[Dict] = None,
     ) -> None:
+        """
+        A configuration space organizes all hyperparameters and its conditions
+        as well as its forbidden clauses. Configurations can be sampled from
+        this configuration space.
+        
+        Parameters
+        ----------
+        name : str
+            Name of the configuration space
+        seed : int
+            random seed
+        meta : dict 
+            metadata (not used)
```

Comment:
Could you please replace the `(not used)` by something more possitive like `field for holding meta data provided by the user, not used by the configuration space`.